core/ParsedArg.h

```diff
@@ -9,13 +9,16 @@ namespace sorbet::core {
 struct ParsedArg {
     struct ArgFlags {
         bool isKeyword : 1;
-        bool isRepeated : 1;
+        bool isRepeated : 1; // *args or **kwargs
         bool isDefault : 1;
         bool isShadow : 1;
         bool isBlock : 1;
+        bool isKwSplat : 1; // only **kwargs
```

Comment:
I don't think we need this. It should already be the case that you can tell `isKwSplat` from `isKeyword && isRepeated`.